package-lock.json

```diff
@@ -1,8 +1,16634 @@
 {
```

Comment:
Should we be making changes to this? Maybe npm install is mutating this file? Ref: https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore